prov/sockets/src/sock.h

```diff
@@ -135,6 +135,11 @@
 #define SOCK_EP_COMP_ORDER (FI_ORDER_STRICT | FI_ORDER_DATA)
 #define SOCK_EP_DEFAULT_OP_FLAGS (FI_TRANSMIT_COMPLETE)
 
+#define SOCK_EP_CQ_FLAGS (FI_SEND | FI_TRANSMIT | FI_RECV | \
```

Comment:
FI_SEND and FI_TRANSMIT are the same value.  I merged this request, but for clarity, I would drop FI_SEND, since FI_TRANSMIT is what is called out in the man page.